src/Views/student-signin/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-grid-system';
+import { Card } from 'react-rainbow-components/components';
+import styled from 'styled-components';
+import StandardInput from '../../atoms/forms/textinput-field';
+import ModuletteButton from '../../atoms/standard-button';
+import NavbarStyle1 from '../../components/navbar/style1'
```

Comment:
Change imports to be with uppercase `A` and `C` or change folder names to be with lowercase `a` and `c`